.github/workflows/greetings.yml

```diff
@@ -0,0 +1,32 @@
+# IMPORTANT NOTE
+# In this workflow there should NOT be checkout action - because of security reasons.
+# More info:
+# https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target
+# https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
+
+name: PR welcome message
+on:
+  pull_request_target:
+    types: [opened]
+
+jobs:
+  pr_comment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create comment
+        uses: peter-evans/create-or-update-comment@v2
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            ## **Thank you for contributing to the Convert2RHEL project!**
+            👋 Hello @${{ github.actor }}, thank you for submitting a PR 🚀!
+            Please note that every PR needs to comply with the [Convert2RHEL Guidelines](https://github.com/oamg/convert2rhel/blob/main/CONTRIBUTING.md) and must pass all tests in order to be mergable.
+            If you want to rebuild a package in copr, you can use following commands as a comment:
+            - **/packit copr-build** to submit a public copr build using packit
+
+            To launch regression testing public members of oamg organization can leave the following comment:
+            - **/rerun** to schedule regression tests using this pr build
+
+            Please [open ticket](https://github.com/oamg/convert2rhel/issues) in case you experience technical problem with the CI.
+
+            **Note:** In case there are problems with tests not being triggered automatically on new PR/commit or pending for a long time, please tag @oamg/convert2rhel-developers in the pr comments.
```

Comment:
@r0x0d I know this was created and merged during my PTO so I missed the conversation. But is the goal of this workflow? What issue did it solve?

Reply:
It is supposed to warn the user that opened a PR how our workflow for integration tests should work. 

In this case, this is mostly targeted to external contributors to warn them that integration tests will not run as normally expected, but, we will trigger that if we think the PR is safe to do so.

Reply:
Yeah, the main goals was to pass this message _"If you're an external collaborator (Meaning that you're not part of oamg), ping the `@oamg/convert2rhel-developers` group to assign the testing label to your pr."_

Even though, as I'm thinking about it, maybe we shouldn't be telling them to ping us. We should know what to do. And if somebody creates a PR for us, we want the tests to pass, so we will add the label.

So I'd change it to:
_If you're an external collaborator (meaning that you're not part of the oamg organization), wait for somebody from the `@oamg/convert2rhel-developers` group to add the `testing` label to the PR to trigger the tests._

Reply:
Changed the contents of the message.